.github/workflows/tests.yml

```diff
@@ -10,13 +10,122 @@ on:
     - cron: '0 */2 * * *'
 
 jobs:
+  build_docker_faucet:
```

Comment:
there's a lot of redundancy in these jobs, can they be combined into a matrix?

Reply:
Changed to a matrix.